packages/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.13.1
```

Comment:
It seems like this should be a patch (+) change since this plugin is pre-1.0 and for consistency with the other plugins you're changing.

Reply:
Done.